src/System.Private.ServiceModel/tests/Scenarios/Infrastructure/SetupValidationTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using Infrastructure.Common;
+using System;
+using Xunit;
+
+public partial class SetupValidationTests
+{
+    [Fact]
+    [OuterLoop]
+    public static void Root_Certificate_Available_From_Server()
+    {
+        // *** SETUP *** \\
+        byte[] certData = null;
+        Exception exception = null;
+
+        // *** EXECUTE *** \\
+        try
+        {
+            certData = ServiceUtilHelper.GetRootCertificateDataFromServer();
+        }
+        catch (Exception e)
+        {
+            exception = e;
+        }
+
+        // *** VALIDATE *** \\
+        Assert.True(exception == null,
+                    String.Format("The root certificate could not be obtained from the server{0}{1}",
+                                  Environment.NewLine,
+                                  exception == null ? String.Empty : exception.ToString()));
+
+        Assert.True(certData != null, "The server returned a null root certificate");
+        Assert.True(certData.Length > 0, "The server returned an empty root certificate");
```

Comment:
Not blocking for merge -- 
Do we want to verify the certificate as well, such as opening up the certificate to see that it's got a valid cert? No need to validate contents of the cert (e.g. subject name), but we might want to use X509Certificate2 to validate that the cert was correctly retrieved and that it's a valid cert. 

... perhaps we can figure out from here that the certificate is expired? :smile: